### ISSUE TITLE
feat: ability to report creation for harmful or abusive content

### ISSUE THREAD

**Issue by Luckey-Elijah** [Luckey-Elijah/falling_sand#18]
I understand this in some ways might be an ethically (or a morally) gray area, but there should be someway to review deal with this in some capacity

**Comment by Luckey-Elijah** [comment]
table maybe would look like
```sql
CREATE TABLE reports (
    report_id INTEGER PRIMARY KEY AUTOINCREMENT,
    creation_id INTEGER NOT NULL,
    resolved BOOLEAN NOT NULL DEFAULT 0,
    message TEXT NOT NULL,
    category TEXT NOT NULL,
    FOREIGN KEY (creation_id) REFERENCES creations(id)
);
```

**Comment by ClementBeal** [comment]
You could add those fields :

```sql
CREATE TABLE reports (
... as before
    creation_date DATETIME NOT NULL,
     category_id
    FOREIGN KEY (category_id) REFERENCES report_categories(id)
);

CREATE TABLE report_categories {
    id INTEGER PRIMARY KEY,
    name TEXT NON NULL,
}
```

**Comment by Luckey-Elijah** [comment]
closed since [72f443d](https://github.com/Luckey-Elijah/falling_sand/commit/72f443d7c1f0ad49ef97ed23d0adb8332e5f2090)
